README.md

```diff
@@ -15,3 +15,5 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+The worst case scenario would be if the array was reversely sorted (ie. the greatest element in front with the following descending). In this case, the outer loop increases the subarray sizes by multiples of two, menaing it takes $log_{2}(n)$ time. The inner loop visits every element of the array, meaning that it will take $n$ time. Thus, the worst-case time complexity of this mergesort is $O(nlog(n))$. 
```

Comment:
Hmm, isn't your implementation doing more work to keep in in place?

Reply:
Well wouldn’t it be the same as the recursive version since that one splits the arrays into two until it gets to individual elements, then since it’s worst case it will have to visit every element in the array?

Reply:
What about the merge step? Are you doing more work there to move elements to the correct place?

Reply:
So does that make the worst case time complexity $O(n^{2}log(n))$ then?

Reply:
Sounds like it's going in the right direction :)